tests/containers/nodejs/matrix.Dockerfile

```diff
@@ -0,0 +1,13 @@
+ARG NODE_IMAGE_TAG
```

Comment:
I suppose we could use the same Dockerfile? This can be the only version we maintain (named `Dockerfile`), and we just need to add the `node@sha256:59531d2835edd5161c8f9512f9e095b1836f7a1fcb0ab73e005ec46047384911` as a build arg to the `""` and `"without-flags"` images.